### ISSUE TITLE
publish 2.0.0-alpha.3 to capture https://github.com/dart-lang/webdev/pull/279

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/webdev#281]
We're seeing issues on the CI for dart devtools to do with webdev not supporting build_web_compilers 2.0.0-alpha.0 (https://travis-ci.org/flutter/devtools/jobs/519802635#L317):

```
webdev • webdev could not run for this project.
webdev • The `build_web_compilers` version – 2.0.0-alpha.0 – is not within the allowed constraint – >=0.3.6 <2.0.0.
```

I think this was addressed by https://github.com/dart-lang/webdev/pull/279, so likely a publish of 2.0.0-alpha.3 would fix the devtools CI.

**Comment by jacob314** [comment]
fyi @jakemac53

**Comment by terrylucas** [comment]
Gary just published dev alpha 3.

Thanks Gary.

**Comment by jakemac53** [comment]
For some reason it looks like an `any` dependency constraint on `build_web_compilers` is picking up the `-alpha.0` release which was not expected.

The fix for anybody blocked by this is to change your constraint on `build_web_compilers` to `^1.0.0`.

**Comment by jakemac53** [comment]
fyi @natebosch

**Comment by jakemac53** [comment]
Alternatively you can global activate the absolute latest webdev, but you will have to very explicitly do that with `pub global activate webdev 2.0.0-alpha.3`

**Comment by terrylucas** [comment]
This is a temporary work around right?  It's a breaking change that 'any', in a normal pubspec, for build_web_compilers causes webdev serve to fail.

**Comment by jakemac53** [comment]
Once a non-alpha webdev is released then a pub global activate will get you a version that works. Unfortunately we have no mechanism to make globally activated packages and locally depended on packages coordinate in terms of the version solve, so there really isn't anything we can do.

Fwiw, we did do a breaking change release in build_web_compilers and so we are following semver, the `any` dependency is the problem.

**Comment by jakemac53** [comment]
Going to close this as a compatible version of `webdev` has been released (2.0.0-alpha.3)
